docs/proposals/PROPOSAL_E.md

```diff
@@ -157,6 +163,51 @@ If a query results in no referrers found, an empty manifest list MUST be returne
 - The registry SHOULD return fewer than `n` results when the generated Index would exceed the recommended maximum manifest size.
 - The `Link` HTTP header is included in the response when additional results are available and is set to the URL for the next page of results.
 
+#### Filtering
+A limited ability to filter results based on `artifactType` will enable the referrers API to scale as the number of artifact use cases grows.
+ - The registry SHOULD allow for filtering based on `artifactType`.
+ - Clients MAY request results be filterd by adding the `artifactType` query parameter to requests to the referrers API.
+ - The value for `artifactType` MUST be the IANA media type of the artifact to be returned.
```

Comment:
```suggestion
 - The value for `artifactType` MUST be the IANA media type of the artifact to be returned.
 - Clients SHOULD send only one `artifactType` query parameter, and registries SHOULD only interpret one when filtering, if a client sends multiple values.
```

Not totally sold on the wording, but I want to get out ahead of questions about filtering to include `A or B`